test/e2e/e2e.go

```diff
@@ -156,7 +154,6 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 var _ = ginkgo.SynchronizedAfterSuite(func() {
 	// Run on all Ginkgo nodes
 	framework.Logf("Running AfterSuite actions on all nodes")
-	framework.RunCleanupActions()
```

Comment:
could you point out the commit in the test setup in k/k where this was removed please?

Reply:
https://github.com/kubernetes/kubernetes/commit/fc092eb1455f2eb055258a7249b16a63e3a1d7f4